yarn-project/noir-compiler/src/index.test.ts

```diff
@@ -41,7 +42,7 @@ describe('noir-compiler', () => {
     });
 
     it('compiles the test contract', () => {
-      expect(compiled).toMatchSnapshot();
+      expect(compiled.map(withoutDebug)).toMatchSnapshot();
```

Comment:
This actually embedded my local path in the snapshot so it was failing in CI. The only reason it was passing before was because this test wasn't being run in CircleCI (no nargo there).